app/controllers/api/v1/airlines_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::AirlinesController < ApplicationController
+
+  def show
+    render json: Airline.find(params[:id])
+
+    @id = params[:id]
```

Comment:
I don't believe this line is necessary -- this is an API controller and you're sending back the data for your found airline. Please delete (unless there's a purpose I'm missing)